rainloop/lib/Controller/AjaxController.php

```diff
@@ -0,0 +1,95 @@
+<? php
+
+namespace OCA\RainLoop\Controller;
+
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\JSONResponse;
+use OCP\IRequest;
+
+class AjaxController extends Controller {
```

Comment:
@pierre-alain-b: This controller is supposed to handle what was previously handled by the PHP files in the old `ajax/` directory. I essentially created this controller, and the appropriate routes in `appinfo/routes.php`, and then dumped the contents of those files into each respective function below. However, I wasn't sure what the functions were even used for so wasn't sure how to test that they are working as expected. Do you have any insights into how I can test that these replicate the functionality of the old files in `ajax/`?